pixeldata/src/lib.rs

```diff
@@ -103,7 +99,7 @@ pub enum Error {
     InvalidImageBuffer,
 
     #[snafu(display("Unknown GDCM error while decoding image"))]
-    UnknownGdcmError { source: gdcm_rs::Error },
+    UnknownGdcmError,
```

Comment:
At this point, I believe this would call for an opaque error source, so that the error type does not depend on the implementation of pixel data encoders and decoders. Could you experiment and see if something like this would work?

```suggestion
    ImplementerError { source: Box<dyn std::error::Error + Send + 'static> },
```

Reply:
Yes, that would make sense to make it an opaque error. I have changed it!